src/test/unit_tests/test_nsfs_access.js

```diff
@@ -40,13 +43,20 @@ mocha.describe('new tests check', async function() {
         backend: '',
         warn_threshold_ms: 100,
     };
+
+    const NON_ROOT3_FS_CONFIG = {
+        uid: 1574,
+        gid: 1574,
+        backend: '',
+        supplemental_groups: [1572, 1577], //gid of non-root1 and unrelated gid
+        warn_threshold_ms: 100,
+    };
     mocha.before(async function() {
         if (test_utils.invalid_nsfs_root_permissions()) this.skip(); // eslint-disable-line no-invalid-this
-        await fs_utils.create_fresh_path(p, 0o770);
+        await fs_utils.create_fresh_path(p, 0o777);
```

Comment:
Why do you change the permission?

Reply:
same reason as the comment above. for my tests I need root to have full permissions. otherwise non root users cant create new directories